lasagne/layers/base.py

```diff
@@ -1,5 +1,7 @@
 from collections import OrderedDict
 
+from theano import tensor as T
+
```

Comment:
That one also needs to be removed again. You didn't change anything else in `base.py` that requires this.